src/MissionManager/MavCmdInfoCommon.json

```diff
@@ -509,8 +509,8 @@
             "category":     "Flight control",
             "param1": {
                 "label":            "Type",
-                "enumStrings":      "Airspeed,Ground Speed",
-                "enumValues":       "0,1",
+                "enumStrings":      "Airspeed,Ground Speed,Ascend Speed, Descend Speed",
```

Comment:
There should not be spaces between the individual enum string values. Not the extra space after the comma. It might work fine, but would have to look at the code to determine. Better to remove.